osmosis/assetlist.json

```diff
@@ -475,6 +475,46 @@
           }
         }
       ]
+    },
+    {
+      "description": "Membrane's CDP-style stablecoin called CDT",
+      "denom_units": [
+        {
+          "denom": "factory/osmo1s794h9rxggytja3a4pmwul53u98k06zy2qtrdvjnfuxruh7s8yjs6cyxgd/ucdt",
+          "exponent": 0
+        },
+        {
+          "denom": "cdt",
+          "exponent": 6
+        }
+      ],
+      "base": "factory/osmo1s794h9rxggytja3a4pmwul53u98k06zy2qtrdvjnfuxruh7s8yjs6cyxgd/ucdt",
+      "name": "CDT",
+      "display": "cdt",
+      "symbol": "CDT"
+      "logo_URIs": {
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/CDT.svg"
```

Comment:
I think this is still missing

Reply:
Sorry about that, I swear I uploaded both together. Just reuploaded.